modules/status-roles/events/presenceUpdate.js

```diff
@@ -0,0 +1,22 @@
+const {localize} = require("../../../src/functions/localize");
+
+module.exports.run = async function (client, newPresence) {
```

Comment:
```suggestion
module.exports.run = async function (client, oldPresence, newPresence) {
```
I belive you use the oldPresence object currently, instead of the new presence object, please consult the [discord.js docs](https://discord.js.org/#/docs/discord.js/stable/class/Client?scrollTo=e-presenceUpdate)

Reply:
Where am I using oldPresence? 🤔 Afaik what I wrote just depends on what is in the Presence after the change

Reply:
No, the order is relevant, not the name. As describe in the documentation, `oldPresence` is passed first, while ´newPresence` is the secound parameter (here third because we add `client`).
With the current code, users may have the role even when they have changed their status.

Reply:
oh. that makes sense. my bad

Reply:
![image](https://user-images.githubusercontent.com/43812987/166561254-6d2f5463-6304-4be2-baa6-6b4cedb6ddd6.png)
![image](https://user-images.githubusercontent.com/43812987/166561382-d39ec3e4-a62d-4318-8015-cc3cd0cb7c50.png)